examples/src/main/java/ai/djl/examples/training/transferlearning/TransferFreshFruit.java

```diff
@@ -0,0 +1,195 @@
+        }
+
+        // Also available at
+        // "https://mlrepo.djl.ai/model/cv/image_classification/ai/djl/pytorch/resnet18_embedding/0.1/traced_resnet18_embedding.pt.gz";
+        String modelUrls = "/Users/fenkexin/Desktop/transferDJL/code/base_nw.pt";
+        Criteria<NDList, NDList> criteria =
+                Criteria.builder()
+                        .setTypes(NDList.class, NDList.class)
+                        .optModelUrls(modelUrls)
+                        .optEngine(Engine.getDefaultEngineName())
+                        .optProgress(new ProgressBar())
+                        .optOption("retrain", arguments.isPreTrained() ? "0" : "1")
+                        .build();
+
+        ZooModel<NDList, NDList> embedding = criteria.loadModel();
+
+        Block baseBlock = embedding.getBlock();
+        Block blocks =
+                new SequentialBlock()
+                        .add(baseBlock)
+                        .addSingleton(nd -> nd.squeeze(new int[] {2, 3}))
+                        .add(Linear.builder().setUnits(2).build()) // linear on which dim?
+                        .addSingleton(nd -> nd.softmax(1));
+
+        Model model = Model.newInstance("TransferFreshFruit");
+        model.setBlock(blocks);
+
+        // Config trainer
+        DefaultTrainingConfig config = setupTrainingConfig(arguments);
+
+        // Customized learning rate
+        float lr = 0.001f;
+        FixedPerVarTracker.Builder learningRateTrackerBuilder =
+                FixedPerVarTracker.builder().setDefaultValue(lr);
+        for (Pair<String, Parameter> paramPair : baseBlock.getParameters()) {
+            learningRateTrackerBuilder.put(paramPair.getValue().getId(), 0.1f * lr);
+        }
+        Optimizer optimizer =
+                Adam.builder().optLearningRateTracker(learningRateTrackerBuilder.build()).build();
+        config.optOptimizer(optimizer);
+
+        // Config trainer
+        Trainer trainer = model.newTrainer(config);
+        trainer.setMetrics(new Metrics());
+
+        // Initialize the parameter shape and value
+        int batchSize = 32;
+        Shape inputShape = new Shape(batchSize, 3, 224, 224);
+        trainer.initialize(inputShape);
+
+        // Data
+        ImageFolder datasetTrain = getData("test", "banana", batchSize);
+
+        // Train
+        EasyTrain.fit(trainer, 6, datasetTrain, null);
+
+        // Save model
+        // model.save("your-model-path");
+
+        model.close();
+        embedding.close();
+        return null;
+    }
+
+    private static ImageFolder getData(String subfolderName, String fruit, int batchSize)
+            throws TranslateException, IOException {
+        // The dataset is from <a
+        // href="https://www.kaggle.com/datasets/sriramr/fruits-fresh-and-rotten-for-classification">https://www.kaggle.com/datasets/sriramr/fruits-fresh-and-rotten-for-classification</a>
+        String folderUrl = "/Users/fenkexin/Desktop/transferDJL/code/data/" + fruit;
```

Comment:
Can we add this dataset int our repo?
We should not hard code local path

Reply:
Should we put it in S3 or in our repo. The training dataset can be big.